Index: app/src/main/res/layout/trip_action_bar.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<RelativeLayout xmlns:android=\"http://schemas.android.com/apk/res/android\"\r\n    android:layout_width=\"match_parent\"\r\n    android:layout_height=\"match_parent\">\r\n\r\n    <TextView\r\n        android:id=\"@+id/tripDay\"\r\n        android:layout_width=\"wrap_content\"\r\n        android:layout_height=\"wrap_content\"\r\n        android:text=\"Coucou\" />\r\n</RelativeLayout>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/layout/trip_action_bar.xml b/app/src/main/res/layout/trip_action_bar.xml
--- a/app/src/main/res/layout/trip_action_bar.xml	(revision 2f31db84ba95785fe362cd86bbe5ae5b4d864272)
+++ b/app/src/main/res/layout/trip_action_bar.xml	(date 1646245726214)
@@ -7,5 +7,5 @@
         android:id="@+id/tripDay"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Coucou" />
+        android:text="Les voyages du " />
 </RelativeLayout>
\ No newline at end of file
